### PR TITLE
Disable button until a symptom has been checked

### PR DESCRIPTION
**Pull request by Sam-gtri**
Disables the submit button until a symptom is selected to prevent empty submissions

### CODE REVIEW

**Comment by Sam-gtri** [comment]
@scottleibrand there already is a "none of the above" checkbox

File: CoEpi/ui/healthquiz/HealthQuizViewController.swift [hunk 1]
```diff
@@ -37,7 +39,8 @@ class HealthQuizViewController: UIViewController, ErrorDisplayer {
 
     override func viewDidLoad() {
         questionList.register(cellClass: QuestionCell.self)
-
+        submitButton.isEnabled = false
```
Comment (i-schuetz) [inline CoEpi/ui/healthquiz/HealthQuizViewController.swift:42]: This should be reactive. Add an observable to the view model with enabled status. Derive enabled status from items observable (`selectedSymptoms` is not empty).

File: CoEpi/ui/healthquiz/HealthQuizViewController.swift [hunk 2]
```diff
@@ -17,6 +18,7 @@ class HealthQuizViewController: UIViewController, ErrorDisplayer {
         
         dataSource.onChecked = { (question, idx) in
             viewModel.handleAnswer(question: question, idx: idx)
+            self.submitButton.isEnabled = true
```
Comment (i-schuetz) [inline CoEpi/ui/healthquiz/HealthQuizViewController.swift:21]: If the boxes are unchecked, the button would stay enabled.

**Comment by scottleibrand** [comment]
Ok, as long as checking None of the Above still allows the report to be submitted, I withdraw my objection. Thanks.

**Pull request merged**